examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -957,6 +957,24 @@ client cluster OnOff = 6 {
   attribute enum8 startUpOnOff = 16387;
   readonly global attribute bitmap32 featureMap = 65532;
   readonly global attribute int16u clusterRevision = 65533;
+
+  request struct OffWithEffectRequest {
```

Comment:
This is remaining non-determinism, right?  I assume this will be fixed in a followup?

Reply:
I have no idea what this file is and why this changed. @tecimovic do you know what this file is and why this is changing?

Reply:
This file is generated by ZAP, and presumably it's also using the helpers that are non-deterministic and its template needs to be fixed accordingly.  That can happen in a separate PR.